unittests/wasm-spec-tests/generated-tests/address.cpp

```diff
@@ -11,10 +11,7 @@ BOOST_DATA_TEST_CASE(address_0_check_throw, boost::unit_test::data::xrange(0,11)
    tester.set_code("wasmtest"_n, wasm_address_0);
    tester.produce_block();
 
-   action test;
-   test.account = "wasmtest"_n;
-   test.name = account_name((uint64_t)index);
-   test.authorization = {{"wasmtest"_n, config::active_name}};
+   action test({{"wasmtest"_n, config::active_name}}, "wasmtest"_n, account_name((uint64_t)index), {});
```

Comment:
I guess we found no way to regenerate these files properly? I think if anything should update the generator,
https://github.com/AntelopeIO/spring/blob/f45de1f9f0d034855efdd3d8fdf29c77ec43759d/unittests/wasm-spec-tests/generator/eosio_wasm_spec_test_generator.cpp#L39-L43

btw it sure looks like AntelopeIO/wasm-spec-tests@026fb4b found a way to regenerate.

Reply:
Updated the generator https://github.com/AntelopeIO/spring/pull/951/commits/b158e5b2b187b226e59e58eb64cb735a21f8d696.